src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color, transform 300ms linear;
```

Comment:
1) linear is default transition-timing-function. No need to declare it
2) move 300ms to a variable since it's a repeated value
3) add transition-duration to a color